engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -73,6 +79,10 @@ public String apply(@Nullable Level input)
     }
   };
 
+  static {
```

Comment:
MDC properties are not static. Each thread may have it's own MDC properties populated.